app/views/devise/registrations/edit.html.erb

```diff
@@ -0,0 +1,43 @@
+<h2>Edit <%= resource_name.to_s.humanize %></h2>
```

Comment:
If you dont modify these devise view files you need to remove them

Reply:
removed